Modules/_localemodule.c

```diff
@@ -94,7 +104,8 @@ PyLocale_setlocale(PyObject* self, PyObject* args)
 #if defined(MS_WINDOWS)
     if (category < LC_MIN || category > LC_MAX)
     {
-        PyErr_SetString(Error, "invalid locale category");
+        PyErr_SetString(get_locale_state(self)->Error,
+                                         "invalid locale category");
```

Comment:
your indentation it not correct. "invalid..." should be aligned with get_locale..., not with (self...). Same comment for other changes below.

Reply:
thanks, done.